blog.html

```diff
@@ -0,0 +1,75 @@
+            </div>
+        </div>
+        <div class="blogentry">
+            <img src="images/blog-2.jpg" class="blogpic" alt="Several dogs in park on their leashes appear to look at their walker">
+            <div class="text">
+              <h3>How To Walk Multiple Dogs</h3>
+              <p class="blogtext">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris sed eros elit. Donec nisi ex, varius eget tortor molestie, vulputate aliquet justo. Fusce eu fringilla diam. Suspendisse interdum dictum mauris, eu bibendum nisi ullamcorper ac. Vivamus vestibulum ligula nec urna condimentum, non vehicula quam sagittis. Vestibulum congue varius enim quis elementum. Nulla finibus urna turpis, in pellentesque dui iaculis sit amet. Aliquam placerat neque quis odio venenatis egestas. Quisque fringilla eros eget quam accumsan, quis tempor metus rhoncus. Pellentesque nec lectus ut libero iaculis varius. Fusce nunc mauris, tincidunt vel nulla vitae, dapibus lacinia quam. Integer at facilisis sapien. Etiam nec volutpat dui.</p>
+        
+            </div>
+        </div>
+        <div class="blogentry">
+            <img src="images/blog-3.jpg" class="blogpic" alt="In profile at sunset, a person holds up a ball as a dog wathces">
+            <div class="text">
+              <h3>Teach Your Dog To Fetch</h3>
+              <p class="blogtext">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris sed eros elit. Donec nisi ex, varius eget tortor molestie, vulputate aliquet justo. Fusce eu fringilla diam. Suspendisse interdum dictum mauris, eu bibendum nisi ullamcorper ac. Vivamus vestibulum ligula nec urna condimentum, non vehicula quam sagittis. Vestibulum congue varius enim quis elementum. Nulla finibus urna turpis, in pellentesque dui iaculis sit amet. Aliquam placerat neque quis odio venenatis egestas. Quisque fringilla eros eget quam accumsan, quis tempor metus rhoncus. Pellentesque nec lectus ut libero iaculis varius. Fusce nunc mauris, tincidunt vel nulla vitae, dapibus lacinia quam. Integer at facilisis sapien. Etiam nec volutpat dui.</p>
+            </div>
+          </div>
+    </section>
+    
+    <section id="footer">
+      <div class="basefield">
+        <img src="images/adoptadog-logo.svg" id="bottomlogo" alt= "profile of a dog in white">
+        <p id="footer-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras venenatis molestie leo <br><br></p>
+        <a class="footer-link" href="index.html">Home&nbsp;&nbsp;/</a><a class="footer-link" href="dogs.html">&nbsp;&nbsp;Dogs&nbsp;&nbsp;/</a><a class="footer-link" href="contact.html">&nbsp;&nbsp;Contact</a><br><br>
+        <p id="copyright">© 2018 Adoptadog, LLC | Seattle, WA</p>
```

Comment:
instead of the actual copy right symbol it's better to use &copy;

Reply:
poop...looks like markdown made my text look like the symbol...


```
&copy;
``` 

is what I'm looking for

Reply:
Ha. I thought something like that might be afoot. The two things looked pretty indistinguishable. Thanks for clarifying.